.github/workflows/chain-spec-snapshot-build.yml

```diff
@@ -13,7 +13,7 @@ on:
 
 jobs:
   chains-spec:
-    runs-on: self-hosted-ubuntu
+    runs-on: ${{ '$GITHUB_ACTOR' == 'subspace' && 'self-hosted-ubuntu' || 'ubuntu-22.04' }}
```

Comment:
We already used `repository_owner` what is `$GITHUB_ACTOR`? I don't see it documented anywhere.

Also please don't mix `&&` and `||` on the same level, use `()` to make ordering explicit.